jmx-metrics/docs/target-systems/wildfly.md

```diff
@@ -0,0 +1,76 @@
+# WildFly Metrics
+
+The JMX Metric Gatherer provides built in WildFly metric gathering capabilities.
+
+### Deployment Metrics
+* Name: `wildfly.session.count`
+* Description: The number of sessions created.
+* Unit: `{sessions}`
+* Labels: `deployment`
+* Instrument Type: LongCounterCallback
+
+* Name: `wildfly.session.active`
+* Description: The number of currently active sessions.
+* Unit: `{sessions}`
+* Labels: `deployment`
+* Instrument Type: LongUpDownCounterCallback
+
+* Name: `wildfly.session.expired`
+* Description: The number of sessions that have expired.
+* Unit: `{sessions}`
+* Labels: `deployment`
+* Instrument Type: LongCounterCallback
+
+* Name: `wildfly.session.rejected`
+* Description: The number of sessions that have been rejected.
+* Unit: `{sessions}`
+* Labels: `deployment`
+* Instrument Type: LongCounterCallback
+
+### Listener Metrics
+* Name: `wildfly.request.count`
+* Description: The number of requests received.
+* Unit: `{requests}`
+* Labels: `server`, `listener`
+* Instrument Type: LongCounterCallback
+
+* Name: `wildfly.request.time`
+* Description: The total amount of time spent on requests.
+* Unit: `ns`
+* Labels: `server`, `listener`
+* Instrument Type: LongCounterCallback
+
+* Name: `wildfly.request.server_error`
+* Description: The number of requests that have resulted in a 500 response.
```

Comment:
is this a bit more accurate?

```suggestion
* Description: The number of requests that have resulted in a 5xx response.
```

Reply:
I agree. That  is more accurate. I've updated the description in the docs, script, and integration test.